pyerrors/correlators.py

```diff
@@ -343,6 +344,36 @@ def GEVP(self, t0, ts=None, sort="Eigenvalue", **kwargs):
         else:
             return reordered_vecs
 
+    def error_gevp(self, t0):
+        """Solves the GEVP with statistical errors.
+        Due to numerical instabilities and sign conventions the result does not necessarily agree with the standard GEVP method.
+
+        Parameters
+        ----------
+        t0 : int
+            The time t0 for the right hand side of the GEVP
+
+        The returned value is in the same form as the one of corr.GEVP(), but the entries are now Obs.
+        """
+        output = [[[] for i in range(self.T)] for j in range(self.N)]
+        G0 = np.vectorize(lambda x: x)(self[t0])
+        chol = cholesky(G0)  # This will automatically report if the matrix is not pos-def
+        chol_inv = inv(chol)
+        for ts in range(self.T):
+            try:
+                Gt = np.vectorize(lambda x: x)(self[ts])
+                new_matrix = chol_inv @ Gt @ chol_inv.T
+                for state in range(self.N):
+                    ev = chol_inv.T @ eigh(new_matrix)[1][:, state]
+                    ev = np.array([e / np.sqrt(ev @ G0 @ ev.T) for e in ev])
+                    [e.gamma_method() for e in ev]  # While this is slow, there is a large risk a user will project without calculating the errors
```

Comment:
Is it a problem if the projection is done without calculating the error? A potential problem with this feature is that one cannot specify any options for the gamma method but performs the error estimation with the default parameters.

Reply:
There is not really a problem with projecting first. But there is a risk that the errors from the vectors are very large and this is overlooked. It is also rather inconvenient to do `[e.gamma_method() for e in vecs[state][ts]]` for all states and `ts` we care about. But, I think I agree, this line can just be removed.

Reply:
The question is maybe what your main objective is. Do you want to look at the errors of the vectors? If so we could also have an `auto_gamma` argument as we have for other methods?

Reply:
Yes, that will do fine.